WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Print Shipping Label/PrintShippingLabelViewController.swift

```diff
@@ -6,25 +6,27 @@ import UIKit
 /// Informational links are displayed for printing instructions and paper size options.
 final class PrintShippingLabelViewController: UIViewController {
     @IBOutlet private weak var tableView: UITableView!
-    @IBOutlet private weak var printButton: UIButton!
+    @IBOutlet weak var reprintButton: UIButton!
```

Comment:
Should we continue to use the generic `printButton` name, which is probably more appropriate?

Reply:
I decided to use `reprintButton` because this button is only used for the design where the button is at the bottom of the screen (when reprinting a label). If that's confusing I could definitely change it.

Reply:
Oh got it, now it's more clear. It would be enough to add a comment on why there is this extra button, to avoid confusion. 👍